Makefile

```diff
@@ -278,3 +280,36 @@ ifeq (,$(wildcard $(AWS)))
 	unzip $PWD/bin/awscliv2.zip -d $PWD/bin/
 	$PWD/bin/aws/install
 endif
+
+NILAWAY_VERSION ?= latest
+NILAWAY := $(TOOLS_HOST_DIR)/nilaway-$(NILAWAY_VERSION)
+
+.PHONY: nilcheck
+nilcheck: $(NILAWAY) ## Run nil check against codemake.
+	go list ./... | xargs -I {} -d '\n' $(NILAWAY) \
+		-exclude-errors-in-files $(PWD)/internal/controller/bucket/bucket_backends.go \
+		-exclude-pkgs github.com/linode/provider-ceph/apis/provider-ceph/v1alpha1,github.com/linode/provider-ceph/internal/backendstore \
```

Comment:
nits: I think it would be better to add a comment for the reason why these are excluded